### ISSUE TITLE
Support for Node.js environment and fallback for printing colors in Terminal/Command prompt

### ISSUE THREAD

**Issue by astoilkov** [astoilkov/console-powers#1]
`console.message` should be installable through `npm install console.message` and should fallback to printing colors in the Terminal(Mac) and Command Prompt(Windows).

**Comment by anupsaund** [comment]
+1

**Comment by eddyystop** [comment]
+1

**Comment by jakewhiteley** [comment]
How are you planning on printing colours within Command Prompt? Would you be relying on a third party library to allow ANSI control sequences within CMD?

**Comment by astoilkov** [comment]
Hi @jakewhiteley,

Actually I could extract the logic from third party library and apply it in mine. The implementation for printing colors to the console is easy and straightforward. The only problem is to print the closest color. The CMD supports only limited number of colors and when you provide a custom color like `#12345` this should be translated to its closest supported CMD color.

**Comment by astoilkov** [comment]
9 years later, I will answer this #1 issue 😄.

I decided to start working on this project again as I find it useful and started using it daily. I rewrote the library and now it heavily relies on web console specifics. Currently, if I implement Node.js support, this would mean that it will be almost as writing 2 different implementations for the web and node. This is why I will close this issue. This is a decision that may change in the future as the library evolves.
